src/renderer/msg_filter.rs

```diff
@@ -0,0 +1,74 @@
+//! This module contains logic for filtering windows messages.
+
+use bitflags::bitflags;
+use windows::Win32::UI::WindowsAndMessaging::*;
+
+bitflags! {
+    /// Flags for filtering window messages.
+    #[repr(transparent)]
+    pub struct MessageFilter: u32 {
+        /// Blocks keyboard input event messages.
+        const InputKeyboard = 1u32 << 0;
+        /// Blocks mouse input event message.
+        const InputMouse = 1u32 << 1;
+        /// Blocks raw input event messages.
+        const InputRaw = 1u32 << 2;
+
+        /// Blocks window gain/lose focus event messages.
+        const WindowFocus = 1u32 << 8;
+        /// Blocks window control event messages
+        /// like move, resize, minimize, etc.
+        const WindowControl = 1u32 << 9;
+        /// Blocks window close messages.
+        const WindowClose = 1u32 << 10;
+
+        /// Blocks messages ID from 0 to WM_USER - 1
```

Comment:
Could you please add backticks to the constants here e.g. `WM_USER`? Just for the sake of documentation consistency.